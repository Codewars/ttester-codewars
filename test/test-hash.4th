\ Copyright 2019 nomennescio
s" string hash" describe#{
  s" short strings" it#{
    <{ s" Hello World!" s# -> s" Hello World!" s# }>
    <{ s" Hello Worlds!" s# -> s" Hello Worlds!" s# }>
  }#
  s" failing compares" it#{
    <{ s" Hello World!" s# -> s" Hello Worlds!" s# }>
    <{ s" Hello Worlds!" s# -> s" Hello Worlds! " s# }>
  }#
  s" failing compares with custom messages" it#{
    <{ s" Hello World!" 2dup ." Actual: '" type ." '" s# -> s" Hello Worlds!" 2dup ." , Expected: '" type ." '" s# }>
    <{ s" Hello Worlds!" 2dup ." Actual: '" type ." '" s# -> s" Hello Worlds! " 2dup ." , Expected: '" type ." '" s# }>
  }#
}#
