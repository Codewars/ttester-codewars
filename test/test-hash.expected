
<DESCRIBE::>string hash

<IT::>short strings

<PASSED::>Test Passed

<PASSED::>Test Passed

<COMPLETEDIN::>0.211 ms

<IT::>failing compares

<FAILED::>Expected 1266463317 , got 610024131 

<FAILED::>Expected 1589968543 , got 1266463317 

<COMPLETEDIN::>0.206 ms

<IT::>failing compares with custom messages
Actual: 'Hello World!', Expected: 'Hello Worlds!'
<FAILED::>Expected 1266463317 , got 610024131 
Actual: 'Hello Worlds!', Expected: 'Hello Worlds! '
<FAILED::>Expected 1589968543 , got 1266463317 

<COMPLETEDIN::>0.344 ms

<COMPLETEDIN::>0.864 ms
